ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -601,11 +601,12 @@ let definition_query server (state : State.t) uri position merlin_request =
     match location_of_merlin_loc uri result with
     | Ok s -> Fiber.return s
     | Error message ->
-      let+ () =
-        let message = sprintf "Locate failed. %s" message in
-        State.log_msg server ~type_:Error ~message
-      in
-      None)
+      let message = sprintf "Merlin Locate failed: %s" message in
+      Jsonrpc.Response.Error.raise
+        (Jsonrpc.Response.Error.make
+           ~code:Jsonrpc.Response.Error.Code.InternalError
```

Comment:
I don't think this is the right error code. The error could easily occur due to user error - for example, the project isn't built yet. Does `RequestFailed` make more sense?